recipes/gurkenlaeufer/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class GurkenlaeuferConan(ConanFile):
+    name = "gurkenlaeufer"
+    description = "Simple way to write cucumber tests for your C++ projects"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/paulelsner/gurkenlaeufer"
+    topics = ("testing", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 14
```

Comment:
As far as I can see, the recipe says that it supports c++11? Is there a reason I can't see as to why this is 14?